server/test/test.js

```diff
@@ -78,6 +81,9 @@ describe('/POST products', () => {
         done();
       });
   });
+});
+
+describe('/GET products', () => {
 
   it('it should get a particular product', (done) => {
```

Comment:
You did not test for 404 when the product was not found as we discussed yesterday even for the other endpoints

Reply:
working on this will commit shortly